src/system/sargon_os/sargon_os_profile.rs

```diff
@@ -269,17 +149,10 @@ impl SargonOS {
     /// # Emits
     /// Emits `Event::ProfileSaved` after having successfully written the JSON
     /// of the active profile to secure storage.
-    pub(crate) async fn maybe_validate_ownership_update_profile_with<F, R>(
-        &self,
-        validate_ownership: bool, // should only ever pass `false` from `claim`
```

Comment:
Ownership is not validated on profile save, it is rather validated on the Wallet side when uploading the backup or after doing periodic cloud check